met-api/sample.env

```diff
@@ -124,6 +124,8 @@ DATABASE_TEST_NAME=
 DATABASE_TEST_HOST=
 DATABASE_TEST_PORT=
 
+KEYCLOAK_TEST_BASE_URL="http://localhost:8081"
```

Comment:
So does this mean one needs to run an instance of keycloak locally just to be able to test?

Reply:
No need to run a separate instance of Keycloak. The test utilizes a Docker Compose file, and the Keycloak instance is automatically set up when executing the pytest command locally.

Reply:
Oh okay so if I'm understanding: a limited, temporary instance of keycloak is spun up only for testing?